substratest/factory.py

```diff
@@ -53,9 +58,9 @@ def score(self, y_true, y_pred):
 import substratools as tools
 class TestAlgo(tools.Algo):
     def train(self, X, y, models, rank):
-        return [0, 1]
+        return sum(models) + sum(X)
```

Comment:
It's not very clear from reading this line that we're returning a model. 

What about we return an actual dummy model here, like in the [substratools tests](https://github.com/SubstraFoundation/substra-tools/blob/master/tests/test_algo.py#L39)? i.e. `return {'value': 3}`